narwhals/_pandas_like/series.py

```diff
@@ -75,15 +75,13 @@ def __init__(
         *,
         implementation: str,
     ) -> None:
-        """Parameters
-        ----------
-        df
-            DataFrame this column originates from.
-        """
-
```

Comment:
this is leftover from a _really_ old design

ok, maybe not that old, it just feels that way